src/pages/Sightseeingpage/SightseeingPage.stories.js

```diff
@@ -0,0 +1,18 @@
+import SightseeingPage from './SightseeingPage'
+import place1 from './../../images/sightseeing/Hafen.jpg'
```

Comment:
For `place1` and `place2` why not give them specific names?

Reply:
I remove it, didn´t need them at all: :)